docs/getting-started/data-views-in-sec.asciidoc

```diff
@@ -0,0 +1,37 @@
+[[data-views-in-sec]]
+= {data-sources-cap} in Elastic Security
+
+The active {data-source} determines what data appears throughout {es-sec} on pages with event or alert data.
+{data-sources-cap} are defined by the index patterns they include.
+Only data from the {es} {ref}/documents-indices.html[indices], {ref}/data-streams.html[data streams], or {ref}/alias.html[index aliases] specified in the active {data-source} will appear.
+
+IMPORTANT: Custom indices may not be included in the <<default-data-view-security, default {data-source}>>. Modify it or create a custom {data-source} to include custom indices.
+
+[discrete]
+== Switch to another {data-source}
+
+You can tell which {data-source} is active by clicking the {data-source-cap} button that appears at the upper right of {es-sec} pages that display data, such as Overview, Alerts, Timelines, or Hosts.
+To switch to another {data-source}, click **Choose {data-source}**, select one of the options, and click **Save**.
+
+image::images/dataview-select-gif.gif[gif of opening the data view selection menu]
+
+[discrete]
+== Create or modify a {data-source}:
+
+To modify or delete an existing {data-source} or to define a new one, you need the required permissions.
+To learn more, refer to {apm-app-ref}/data-views.html[{kib} {data-sources-cap}].
+
+[discrete]
+[[default-data-view-security]]
+== The default {data-source}
+
+The first time a user visits Elastic Security within a given {apm-app-ref}/xpack-spaces.html[space], the default {data-source} generates in that space and becomes active.
+
+If you delete the active {data-source} when there are no other defined {data-sources}, the default {data-source} will regenerate and become active upon the refresh of any {es-sec} page in the space, because the app requires a {data-source} to function.
+
+The default {data-source} is defined by the `securitySolution:defaultIndex` setting, which you can modify from the Security Solution section of {kib}'s advanced settings (**Stack Management** > **Advanced Settings** > **Security Solution**):
```

Comment:
I felt like this portion sort of buried the lead/lede, starting with a lot of internal workings before actually explaining what the default data source is. One idea is to invert the order a bit (below, along with a few other tweaks).

I also don't feel like we need to point out that the app requires a data source to function, because there isn't much actionable info from that. It'd be one thing if the user was required to re-create a data source to get it the app to work, but since the app does it automatically it's prob enough to just say that.

```suggestion
The default {data-source} is defined by the `securitySolution:defaultIndex` setting, which you can modify in {kib}'s advanced settings (**Stack Management** > **Advanced Settings** > **Security Solution**).

The first time a user visits {elastic-sec} within a given {apm-app-ref}/xpack-spaces.html[space] in Kibana, the default {data-source} generates in that space and becomes active.

If you delete the active {data-source} when there are no other defined {data-sources}, the default {data-source} will regenerate and become active upon the refresh of any {es-sec} page in the space.
```

Reply:
Love this reorg, thanks 👍 👍 👍